### PR TITLE
simulator: take the store size into consideration

### PR DESCRIPTION
**Pull request by rleungx**
## What have you changed? (required)
This PR takes the store size into consideration in the simulator. 

## What are the type of the changes (required)?
- Improvement (non-breaking change which is an improvement to an existing feature)

## How has this PR been tested (required)?
- Manual tests

### CODE REVIEW

File: pkg/faketikv/node.go [hunk 1]
```diff
@@ -218,3 +218,13 @@ func (n *Node) Stop() {
 	n.wg.Wait()
 	simutil.Logger.Infof("node %d stopped", n.Id)
 }
+
+func (n *Node) decStoreSize(size uint64) {
```
Comment (disksing) [inline pkg/faketikv/node.go]: Could you make the name more specific? I can't tell from the name whether it decreases the available size or the used size.

Comment (rleungx) [reply pkg/faketikv/node.go]: `decStoreSize` here means that decreasing the store size. The available size of the store will increase as well as the used sized will be decreased. And `incStoreSize` is similar.

Comment (disksing) [reply pkg/faketikv/node.go]: em, quote "available size of the store will increase as well as the used sized will be decreased", but the code actually decreases availableSize and increases usedSize. That makes me even more confused...

Comment (rleungx) [reply pkg/faketikv/node.go]: edited

**Pull request merged**